### ISSUE TITLE
CMS builds failing; can't fly without umbrella

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-cms#15813]
## Description

Because things were going so great, otherwise.

![Screenshot 2023-10-24 at 9 34 19 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/1318579/c047e570-59e4-4c13-8b2a-43d893a9bf9a)

[Context](https://dsva.slack.com/archives/C03LFSPGV16/p1698149848515659)

```
00:08:58  TASK [common/promtail : Copy init service file.] *******************************
00:08:58  Tuesday 24 October 2023  04:08:58 +0000 (0:00:00.492)       0:03:51.972 ******* 
00:09:01  fatal: [ec2-160-1-126-167.us-gov-west-1.compute.amazonaws.com]: FAILED! => changed=false 
00:09:01    checksum: 55c4b5afceda065764fdced600cf8e2359997e0e
00:09:01    msg: Destination directory /etc/init does not exist
```

**TL;DR**: A promtail step is trying to copy a file to `/etc/init`, but that directory does not exist in AL2-encrypted. Gotta ensure it exists.

## Acceptance Criteria

- [ ] CMS builds work.

**Comment by ndouglas** [comment]
Waiting on a review from Platform.

**Comment by ndouglas** [comment]
Closed by https://github.com/department-of-veterans-affairs/devops/pull/13629
